front/app/modules/commercial/analytics/admin/components/ActiveUsersCard/renderTooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
We should really find a way to simplify/standardize tooltips, every time it's so much boilerplate...

Reply:
i agree, so the dynamic thing here is just the content of `Box` and the payload type, right? So maybe this could be a prop for the chart component (Pie, Line, Bar, etc)

Reply:
Yeah or our chart component converts the weird `recharts` object to a `Payload` and calls this function with it... something like that